packages/core/src/components/misc/CredentialCard.tsx

```diff
@@ -79,16 +86,49 @@ const CredentialCard: React.FC<CredentialCardProps> = ({
     }
   }, [credential, brandingOverlay, resolveBundleForCredential])
 
+  // Resolve identifiers and name (supports webvh via agent) for anoncreds credentials
+  useEffect(() => {
+    let isMounted = true;
+    (async () => {
```

Comment:
nit: Avoid using immediately invoked anonymous functions here. They can make the code harder to read, consider a named function within the useEffect

```ts
const asyncEffect = () => {}
asyncEffect()
```